src/project/mesa3d_desktop_intel.rs

```diff
@@ -265,8 +265,8 @@ impl Project for Mesa3DDesktopIntel {
             && (!inc.contains(&path_to_string(&subprojects))
                 || inc.contains(&path_to_string(&subprojects.join("perfetto"))))
     }
-    fn filter_link_flag(&self, _flag: &str) -> bool {
-        false
+    fn filter_link_flag(&self, flag: &str) -> bool {
+        flag == "-Wl,--build-id=sha1"
```

Comment:
Shouldn't it be a real SHA1, not just the string `sha1`?

Reply:
Oh, that string is the kind of build-id requires, not the id itself. Sounds good to me

Reply:
Right, `-Wl,--build-id=sha1` instructs the linker to perform a SHA1 hash of some important bits of the binary and insert that in a `.note` section in the binary itself.

See https://github.com/mattst88/build-id if you're interested in how this works.